cluster/images/obs-csi-plugin/entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+HOST_CMD="nsenter --mount=/proc/1/ns/mnt"
+fileName=obsfs_CentOS7.6_amd64
+
+echo "Starting deploy obs csi-plugin...."
+mkdir -p /dev/csi-tool/
+$HOST_CMD yum install -y openssl-devel fuse fuse-devel
+tar -zxvf /root/$fileName.tar.gz -C /dev/csi-tool/
+$HOST_CMD cp -r /dev/csi-tool/$fileName/. ./
+$HOST_CMD bash install_obsfs.sh
+
+echo "Starting run obs socket server...."
+cp /bin/socket-server /dev/csi-tool/socket-server
+chmod 755 /dev/csi-tool/socket-server
```

Comment:
Does it have execute permissions in the host?

Reply:
socket-server should have the permission to run